src/Sarif.Viewer.VisualStudio.Core/ErrorList/SarifResultTableEntry.cs

```diff
@@ -108,38 +103,27 @@ public SarifResultTableEntry(SarifErrorListItem error)
 
             // Anything that's a bit more complex, we will make a "lazy" value and evaluate
             // it when it's asked for.
+            // In WPF all the content elements including Inlines can only have one parent element.
+            // If the Inlines are binded to multiple elements, only 1 can show the Inlines.
+            // Create duplicated message inline objects for ErrorList table entry since
+            // Error.MessageInlines is already binded to an element in SARIF explorer.
             this.columnKeyToContent[StandardTableKeyNames2.TextInlines] = new Lazy<object>(() =>
-            {
-                string message = this.Error.RawMessage;
-                List<Inline> inlines = SdkUIUtilities.GetMessageInlines(message, this.ErrorListInlineLink_Click);
-
-                if (inlines.Count > 0)
-                {
-                    return inlines;
-                }
-
-                return null;
-            });
+                this.Error.MessageInlines?.Any() == true ?
+                SdkUIUtilities.GetMessageInlines(this.Error.RawMessage, this.Error.MessageInlineLink_Click) :
+                null);
```

Comment:
wrap using the following format:
condition
? true
: false

i think VS suggests this wrap method.

Reply:
apply this strategy for the following two conditions below.

Reply:
updated all conditions format